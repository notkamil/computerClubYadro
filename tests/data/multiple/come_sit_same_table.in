3
08:00 20:00
10
09:00 1 client2
10:00 1 client1
11:00 2 client1 1
11:00 2 client2 1
12:00 4 client1
13:00 4 client2