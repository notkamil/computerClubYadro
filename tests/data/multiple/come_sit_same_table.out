08:00
09:00 1 client2
10:00 1 client1
11:00 2 client1 1
11:00 2 client2 1
11:00 13 PlaceIsBusy
12:00 4 client1
13:00 4 client2
20:00
1 10 01:00
2 0 00:00
3 0 00:00